packages/cli/src/playground/src/pages/scorers/scorer/index.tsx

```diff
@@ -177,17 +26,28 @@ export default function Scorer() {
     }) || [];
   const isLoading = isLoadingScorer || isLoadingAgents;
   const [filteredByEntityId, setFilteredByEntityId] = useState<string>('all');
+  const [selectedScore, setSelectedScore] = useState<any>(null);
+  const [detailsIsOpened, setDetailsIsOpened] = useState<boolean>(false);
 
-  const [visiblePrompt, setVisiblePrompt] = useState<string>('');
+  // if (scorer) {
```

Comment:
current scorers on the example code does not have propts, so I added it manually, just to test UI code